rotkehlchen/tests/utils/history.py

```diff
@@ -817,9 +817,9 @@ def check_result_of_history_creation(
             assert asset_movements[12].category == AssetMovementCategory.WITHDRAWAL
             assert asset_movements[12].asset == A_BTC
 
-        tx_events = [x for x in events if isinstance(x, HistoryBaseEntry) and x.serialized_event_identifier.startswith('0x')]  # noqa: E501
+        tx_events = [x for x in events if isinstance(x, EvmEvent) and x.serialized_event_identifier.startswith('0x')]  # noqa: E501
         gas_in_eth = FVal('14.36963')
-        assert len(tx_events) == 6
+        assert len(tx_events) in {6, 7}  # can be 7 if there is extra unicode-test event provided
```

Comment:
Is it random?

Reply:
No, is not random. I thought that it will be hard to provide an argument here and check on a specific value, but it ended up not being easy.

Now changed this. Thanks for pointing out.